docs/sources/flow/config-language/expressions/types_and_values.md

```diff
@@ -88,6 +88,31 @@ The supported escape sequences are as follows:
 | `\uNNNN` | A Unicode character from the basic multilingual plane (NNNN is four hexadecimal digits) |
 | `\UNNNNNNNN` | A Unicode character from supplementary planes (NNNNNNNN is eight hexadecimal digits) |
 
+## Raw strings
+
+Raw strings are represented by sequences of Unicode characters surrounded by backticks ``` `` ```. 
```

Comment:
What did we intend to show here with the multiple backticks? I don't think it renders as we wanted it to.

Reply:
This mirrors the description of `Strings` above.

<img width="719" alt="image" src="https://github.com/grafana/agent/assets/39704712/7772da3b-ffce-4eab-ae09-989d0f97e9d1">

<img width="719" alt="image" src="https://github.com/grafana/agent/assets/39704712/c959e4e1-5645-4235-a964-847247c4fe0f">

Reply:
Ah, ok, didn't realize Hugo would render it correctly, thanks!